workflows/pipe-common/shell/sge_setup_master

```diff
@@ -60,6 +60,20 @@ configure_global() {
     sed -i "/reporting_params/c\reporting_params $_SGE_REPORTING_PARAMS" ./global
     sed -i '/gid_range/c\gid_range 20000-30000' ./global
 
+    # Override default mail program for SGE
+    if check_cp_cap CP_CAP_GRID_ENGINE_NOTIFICATIONS; then
+        # configure local configuration
+        qconf -sconf $HOSTNAME > ./$HOSTNAME
+        if [ "$?" -eq "0" ]; then
+            sed -i "/mailer/c\mailer $COMMON_REPO_DIR/shell/pipe_mail" ./$HOSTNAME
+            qconf -Mconf ./$HOSTNAME
+        else
+            echo "mailer $COMMON_REPO_DIR/shell/pipe_mail" > ./$HOSTNAME
+            qconf -Aconf ./$HOSTNAME
+        fi
+        rm -f ./$HOSTNAME
+    fi
```

Comment:
Let's configure only global host here and use some universal paths like `LOCAL_COMMON_REPO_DIR`.

```bash
LOCAL_RUN_DIR="/runs-local/pipeline"
LOCAL_COMMON_REPO_DIR="$LOCAL_RUN_DIR/CommonRepo"
ln -s "$RUN_DIR" "$LOCAL_RUN_DIR"
```

Reply:
After discussion:
 - linking only COMMON_REPO_DIR since it can be overridden in a first place, and linking of RUN_DIR to some common location won't solve a problem
 - using `COMMON_REPO_DIR_MUTUAL_LOC` env to store path to mutual location of CommonRepo dir (default value is `/usr/local/CommonRepo`)